hat/assets/js/apps/Iaso/domains/users/components/Filters.js

```diff
@@ -124,14 +124,27 @@ const Filters = ({ baseUrl, params }) => {
         <Grid container spacing={2}>
```

Comment:
Shouldn't this file be only in #1543? https://github.com/BLSQ/iaso/pull/1543/files#diff-ccea35154e8e5addc9406224be919591e706ee788f0b33c25084e1234afbf295

Reply:
Sorry, my mistake, I think I mixed up some PRs.